WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SiteCreationDomainLoaderFragment.java

```diff
@@ -73,7 +73,7 @@ public void onDestroy() {
     public void load(String keywords) {
         postUpdate(new DomainSuggestionEvent(DomainUpdateStep.UPDATING, keywords, null));
 
-        SiteStore.SuggestDomainsPayload payload = new SiteStore.SuggestDomainsPayload(keywords, true, false, 20);
+        SiteStore.SuggestDomainsPayload payload = new SiteStore.SuggestDomainsPayload(keywords, true, true, false, 20);
```

Comment:
The following import can be added to remove the `SiteStore.` from `SiteStore.SuggestDomainsPayload` to be consistent with [`NewBlogFragment.java`](https://github.com/wordpress-mobile/WordPress-Android/blob/c1ff1c8ec4bfd85a4e633813f892747954f66a1c/WordPress/src/main/java/org/wordpress/android/ui/accounts/NewBlogFragment.java#L321).
``` java
import org.wordpress.android.fluxc.store.SiteStore.SuggestDomainsPayload;
```

Reply:
Makes sense, done with b17fbf5 .